lib/data_recency_minutes/main.py

```diff
@@ -104,9 +104,9 @@ async def get_data_freshness(
     if most_recent_ts == 0:
         return None
 
-    data_freshness = round((utc_endtime.timestamp() - most_recent_ts) / 60)
-
-    return data_freshness
+    data_recency_minutes = round((utc_endtime.replace(tzinfo=timezone.utc).timestamp()
```

Comment:
Do you think we should add any comments regarding what this is doing since it's in the feature library?